extensions/attention-stream/src/background.ts

```diff
@@ -254,17 +285,30 @@ async function stateChangeCallback(newState) {
 }
 
 // Initialize the Rally SDK.
-const rally = new Rally({ enableDevMode, stateChangeCallback, rallySite, studyId, storeId, firebaseConfig, enableEmulatorMode });
+const rally = new Rally({
+  enableDevMode,
+  stateChangeCallback,
+  rallySite,
+  studyId,
+  version: browser.runtime.getManifest().version,
```

Comment:
I don't think this is what we want.

We don't want to pass version info _into_ the SDK, we want to be able to read the version _from_ the SDK.

Reply:
e.g. the version info could be exported via a public function from the Rally object? under the hood, the SDK would poll the user study doc and grab (and cache) the version from there.

Reply:
This is extension passing its version which is passed to the web platform. You want this to know which extension version is currently installed by the user.

Reply:
The "latest" version is already present in the /studies database and web platform already reads that.

Reply:
Chatted offline. The code to read user doc from SDK will be added in Ashwin's PR.